app/scripts/styles/_veda-ui-theme-vars.scss

```diff
@@ -20,12 +20,12 @@ $veda-uswds-fontweight-bold: utils.font-weight('bold');
 $veda-uswds-fontweight-semibold: utils.font-weight('semibold');
 
 // COLORS
-$veda-uswds-color-primary-darker: utils.color('primary-darker');
-$veda-uswds-color-secondary: utils.color('secondary');
-$veda-uswds-color-base-dark: utils.color('base-dark');
-$veda-uswds-color-base-darkest: utils.color('base-darkest');
-$veda-uswds-color-base-light: utils.color('base-light');
-$veda-uswds-color-base-ink: utils.color('ink');
+$veda-uswds-color-primary-darker: utils.color('blue-warm-80v');
+$veda-uswds-color-secondary: utils.color('red-50');
+$veda-uswds-color-base-dark: utils.color('gray-cool-60');
+$veda-uswds-color-base-darkest: utils.color('gray-cool-80'); // re-mapped, theme-token 'base-darkest' originally mapped to 'gray-90'
```

Comment:
Following the USWDS default (to some extent) was an intentional choice for this first template instance, but that could not be the case in the future, I'm thinking about the earth.gov work or any customization for, let's say, the Air Quality instance.

To me, it would make sense that we override all default values for colors, typographies, etc. so we don't have to keep track of the changes and have a single configuration file for each instance.

Reply:
@AliceR not necessarily are we deviating from uswds, we are mostly sticking to it with just a very few looking like they are being re-mapped. This file is just us creating references to uswds tokens so we dont have to import `uswds-core` in each component sass file when we want to use their tokens (also nicely organized so you can see all the tokens we use from uswds in one file). if we need to overwrite a specific style in a uswds component but not across uswds components that use the same token, we can just use these vars to make that change inside that specific component's sass file like [here](https://github.com/NASA-IMPACT/veda-ui/blob/main/app/scripts/components/common/page-header/page-header.scss#L18).

When introducing a new uswds component - if they use something we have already re-mapped in the settings file, it should just get that change. Hopefully that helps answer your thoughts.